.github/workflows/coverage.yml

```diff
@@ -0,0 +1,162 @@
+      KEYCLOAK_AUTH_URL: "https://dev.loginproxy.gov.bc.ca/auth/"
+      BCeID_WEB_SERVICE_WSDL: "https://gws1.test.bceid.ca/webservices/Client/V10/BCeIDService.asmx?wsdl"
+      E2E_TEST_STUDENT_USERNAME: student_e2e_test
+      E2E_TEST_STUDENT_PASSWORD: ${{ secrets.TEST_STUDENT_PASSWORD }}
+      E2E_TEST_AEST_BUSINESS_ADMINISTRATORS_USER: ministry-user-aest-business-administrators@e2e-tests
+      E2E_TEST_AEST_BUSINESS_ADMINISTRATORS_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+      E2E_TEST_AEST_OPERATIONS_USER: ministry-user-aest-operations@e2e-tests
+      E2E_TEST_AEST_OPERATIONS_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+      E2E_TEST_AEST_OPERATIONS_ADMINISTRATORS_USER: ministry-user-aest-operations-administrators@e2e-tests
+      E2E_TEST_AEST_OPERATIONS_ADMINISTRATORS_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+      E2E_TEST_AEST_MOF_OPERATIONS_USER: ministry-user-mof-operations@e2e-tests
+      E2E_TEST_AEST_MOF_OPERATIONS_PASSWORD: ${{ secrets.E2E_TEST_PASSWORD }}
+    steps:
+      # Checkout the PR branch.
+      - name: Checkout Target Branch
+        uses: actions/checkout@v3
+      # Generate the coverage report for API
+      - name: Run API E2E tests on Docker
+        working-directory: "./sources/"
+        run: |
+          make e2e-tests-api
+      - name: SIMS API E2E tests Coverage
+        uses: slavcodev/coverage-monitor-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          status_context: "sims-api"
+          comment_context: "E2E SIMS API Coverage Report"
+          coverage_path: "sources/packages/backend/coverage/api/clover.xml"
+          threshold_alert: 1
+          threshold_warning: 50
+          threshold_metric: "lines"
+          comment_footer: false
+  workers-e2e-tests:
+    name: Workflow Workers E2E tests
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_NAME: aest-sims
+      BUILD_ID: ${{ github.event.number }}
+      POSTGRES_PASSWORD: bcgov!
+      BUILD_REF: ${{ github.base_ref }}
+      POSTGRES_DB: sims
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: admin
+      REDIS_PORT: 6379
+      REDIS_PASSWORD: redis
+      REDIS_STANDALONE_MODE: "true"
+      QUEUE_PREFIX: "{sims-local}"
+      API_PORT: 3000
+      APP_PORT: 8080
+      QUEUE_CONSUMERS_PORT: 3001
+      DB_SCHEMA: sims
+    steps:
+      # Checkout the PR branch.
+      - name: Checkout Target Branch
+        uses: actions/checkout@v3
+      # Generate the coverage report for Workflow Workers
+      - name: Run Workflow Workers E2E tests on Docker
+        working-directory: "./sources/"
+        run: |
+          make e2e-tests-workers
+      - name: Workflow Workers E2E tests Coverage
+        uses: slavcodev/coverage-monitor-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          status_context: "workers"
+          comment_context: "E2E Workflow Workers Coverage Report"
+          coverage_path: "sources/packages/backend/coverage/workers/clover.xml"
```

Comment:
Is coverage_format not required here as we use clover.xml?.

![image](https://user-images.githubusercontent.com/54600590/217102367-5d334454-17a3-49aa-a361-6333cba47375.png)

Reply:
Out-of-box it is defined as `auto` and it was able to detected the `clover` format.